packages/filebrowser-extension/src/index.ts

```diff
@@ -284,10 +290,14 @@ const factory: JupyterFrontEndPlugin<IFileBrowserFactory> = {
       return widget;
     };
 
+    const driveName =
+      PageConfig.getOption('collaborative') === 'true' ? 'YDrive' : '';
+
     // Manually restore and load the default file browser.
     const defaultBrowser = createFileBrowser('filebrowser', {
       auto: false,
-      restore: false
+      restore: false,
+      driveName
```

Comment:
This plugin is really annoying.

Would you mind splitting it into two plugins?

One with the current _factory_ that provides `{ createFileBrowser, tracker }`. And another one that depends on the _factory_ to provide the `defaultBrowser`. It will ease replacing it when extracting the RTC logic.